src/shared/tpm2-util.c

```diff
@@ -813,6 +848,10 @@ int tpm2_seal(
                 .size = sizeof(hmac_sensitive.sensitive),
                 .sensitive.data.size = 32,
         };
+        if (pin) {
+                hash_pin(pin, strlen(pin), hmac_sensitive.sensitive.userAuth.buffer);
+                hmac_sensitive.sensitive.userAuth.size = SHA256_DIGEST_SIZE;
+        }
```

Comment:
i think it would be great to erase this memory when done. i.e. under the `finish` label at the end of this function, add an `explicit_bzero_safe()` call on `hmac_sensitive` (or some part of it, though I personally wouldn't bother)

Reply:
Good idea.